code/game/objects/structures/curtains.dm

```diff
@@ -1,3 +1,71 @@
+//
+// Curtain types declaration
+//
+/decl/curtain_kind
```

Comment:
Would you mind splitting the curtain code out into a separate PR? It'll review easier and looks fine to go in independently of the other changes in this PR.

Reply:
Alright, I'll do that in a bit

Reply:
Still haven't gotten around to it. Had more server issues with outreach so..

Reply:
Okay its done!